### ISSUE TITLE
crumb_requester.py - __init__() got an unexpected keyword argument 'user'

### ISSUE THREAD

**Issue by rcbop** [pycontribs/jenkinsapi#611]
##### ISSUE TYPE
 - Bug Report

##### Jenkinsapi VERSION
0.3.6

##### Jenkins VERSION
2.101

##### SUMMARY
I am receiving the following error by running one of the how_to/use_crumbs examples. 

```
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
  File "/Users/rogerio.c.peixoto/Library/Python/2.7/lib/python/site-packages/jenkinsapi/utils/crumb_requester.py", line 15, in __init__
    super(CrumbRequester, self).__init__(*args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'user'
```

##### EXPECTED RESULTS
Should be able to print all jobs in specified Jenkins.

##### ACTUAL RESULTS
The following error:
```
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
  File "/Users/rogerio.c.peixoto/Library/Python/2.7/lib/python/site-packages/jenkinsapi/utils/crumb_requester.py", line 15, in __init__
    super(CrumbRequester, self).__init__(*args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'user'
```

##### USEFUL INFORMATION

python --version
Python 2.7.10

Crumb example code (test_connection.py) :
```
"""
Example of using CrumbRequester - when CSRF protection is enable in Jenkins
"""
from jenkinsapi.jenkins import Jenkins
from jenkinsapi.utils.crumb_requester import CrumbRequester

jenkins = Jenkins('https://jenkins.xxxxxxx.net', user='rogerio.c.peixoto', password='xxxxxx', requester=CrumbRequester('https://jenkins.xxxxxx.net', user='rogerio.c.peixoto', password='xxxxx'))

for job_name in jenkins.jobs:
  print job_name
```

**Comment by rcbop** [comment]
When I changed the parameter name to **username** I got this:

```
Traceback (most recent call last):
  File "test_connection.py", line 8, in <module>
    requester=CrumbRequester('https://jenkins.xxxxxx.net', username='admin', password='xxxxxx'))
  File "/Users/rogerio.c.peixoto/Library/Python/2.7/lib/python/site-packages/jenkinsapi/utils/crumb_requester.py", line 15, in __init__
    super(CrumbRequester, self).__init__(*args, **kwargs)
TypeError: __init__() got multiple values for keyword argument 'username'
```

**Comment by chris-allan** [comment]
Just ran into this myself when updating some of my usage to include CSRF tokens. The example is incorrect and has been fixed in #610.

**Comment by daniel-carrington** [comment]
I got the same errors, myself. That was why I submitted #610 - because I had to reread the constructor arguments a couple of times to see the mistakes. @lechat , can you please merge #610?
